examples/basics/scene/isocurve_for_trisurface.py

```diff
@@ -38,6 +37,8 @@
 # Add a 3D axis to keep us oriented
 scene.visuals.XYZAxis(parent=view.scene)
 
+view.camera = scene.TurntableCamera()
```

Comment:
I love the xy/z scaling in this example, but I overlooked it at first because I thought the scene was 2d / flat. Start with elevation=30 or so?